pkg/onepassword/items.go

```diff
@@ -30,6 +30,14 @@ func GetOnePasswordItemByPath(opConnectClient connect.Client, path string) (*one
 	if err != nil {
 		return nil, err
 	}
+
+	for _, file := range item.Files {
+		_, err := opConnectClient.GetFileContent(file)
```

Comment:
A consequence of this is that all files in the synced item will be fetched from Connect, even if they will not end up being used due to them being "shadowed" by another field with the same name. I don't expect this to be an issue as it is probably going to occur very rarely and will probably only have a small impact on the performance.

Reply:
I think this will be fine but let's keep an eye on this